src/plugins/borrow-plugins/common/util/composeApprovableActions.js

```diff
@@ -19,10 +20,14 @@ export const composeApprovableActions = (...actions: ApprovableAction[]): Approv
       currencyCode,
       nativeAmount
     },
+    unsignedTxs: actions.reduce((txs, action) => [...txs, ...action.unsignedTxs], []),
     approve: async () => {
+      const txs: EdgeTransaction[] = []
       for (const action of actions) {
-        await action.approve()
+        const txs = await action.approve()
+        txs.push(...txs)
```

Comment:
You have a naming conflict in here, so this code doesn't do what you think.

Reply:
Good catch! That's an embarrassing one haha.